### DISCUSSION TITLE
Merge `custom-block-shape` and `editor-theme3`

### DISCUSSION THREAD

**Discussion by just-a-hriday** [ScratchAddons/ScratchAddons#4040] [Discussion]
`custom-block-shape` and `editor-theme3` both customize blocks in the editor - one changes their shape, the other their color.
Aren't they similar enough to merge?

**Comment by mxmou** [comment]
Addons with lots of settings are difficult to use, so separating addons that are related but do different things is usually a good idea. In this case, it also allows each addon to have independent presets.
